Makefile

```diff
@@ -37,7 +37,7 @@ QtBase-clean:
 	@echo [CLN] QtBase
 	@make -s -C ./build/make/ -f QtBase.mk clean -j $(DETECTED_CORES)
 
-Studio:
+Studio: | QtBase
```

Comment:
I don't think this is necessary, please see my comments about removing `-j 4` above.
We can always look into improving build speed/times again, but this is not the right PR for this and the current way to do it is without the manual -j parameter.